hub/auto/unstructured/image_classification.py

```diff
@@ -122,9 +123,11 @@ def structure(  # type: ignore
 
         with ds:
             paths = self._abs_file_paths
+            skipped_files: list = []
             iterator = tqdm(
                 paths,
-                desc='Ingesting "%s"' % self.source,
+                desc='Ingesting "%s" (%i files skipped)'
+                % (self.source.name, len(skipped_files)),
                 total=len(paths),
                 disable=not use_progress_bar,
```

Comment:
for the ingestion tests, can you please set `use_progress_bar` to False for all of them (including kaggle)?